react/components/Heart.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+interface HeartProps {
+    onClick: () => void
+}
+
+const Heart: React.SFC<HeartProps> = ({ onClick }) => {
+  return <div className="z-9999 w2 h2 mt1 ml1 pa3 pointer hover-bg-light-gray" onClick={onClick}>
```

Comment:
What would I use? I've seen this use case across other files on our store, and I couldn't find telling about `div` not being interactive: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md

Reply:
you can't actually tab to a `div` right? unless you explicitly set the `tabIndex`, which would be too much of a hassle to avoid using a more semantic element such as `button` or `a`.